versions.json

```diff
@@ -826,7 +826,7 @@
     "jwt-cpp": {
         "enable_sha": true,
         "commitish": "6dd190d154db29fad071574aa5558f15a70752ad",
-        "version_string": "0.6.99.0",
+        "version_string": "0.7.99.0",
         "license": "MIT",
         "consortium_build_number": "0",
         "package_revision": "4",
```

Comment:
Don't forget to reset the `package_revision`.

Here's the guide for bumping various versions: https://github.com/irods/externals?tab=readme-ov-file#q-what-are-consortium_build_number-and-package_revision-in-versionsjson

Reply:
Updated.